pages/help-center/errors/serialization.mdx

```diff
@@ -1,53 +1,101 @@
+                break
+        except TransientError as te:
+            jitter = random.uniform(0, jitter) * initial_wait_time 
+            wait_time = initial_wait_time * (backoff_factor ** attempt) + jitter
+            print(f"Commit failed on attempt {attempt+1}. Retrying in {wait_time} seconds...")
+            time.sleep(wait_time)
+        except Exception as e:
+            print(f"Failed to execute transaction: {e}")
+            session.close()
+            raise e
+  ```
+
+  Keep in mind that adjusting the `max_retries`, `initial_wait_time`, `backoff_factor`, and `jitter` is important to avoid overloading the system with retries. 
+  For more information on how to handle retries and adjust parameters, please refer to the respective [client documentation](../../client-libraries.mdx) since examples will vary based on different clients.
+
+- **Understand the client you are using**: Neo4j clients in managed transactions, 
+  have built-in logic to automatically retry transactions that fail due to serialization errors.
   There's typically a timeout associated with these retries, after which the
   client will forward the error to the application code. It is important for
   developers to be aware of this and implement additional error handling as
   needed.
-- **Avoid conflicts:** Implement application-level logic to prevent concurrent
-  transactions from modifying the same data points. This strategy can
-  significantly reduce the likelihood of encountering serialization errors.
+
+  Here is an example of Managed API in Python Neo4j Driver:
+
+  ```python
+  def process_chunk_managed_API(query, create_list):
+    driver = GraphDatabase.driver(HOST_PORT, auth=("", ""))
+    with driver.session(max_transaction_retry_time=180.0, initial_retry_delay=0.2, retry_delay_multiplier=1.1, retry_delay_jitter_factor=0.1) as session:
+        session.execute_write(lambda tx: tx.run(query, {"batch": create_list}))
+    driver.close()
+  ```
+
+  Again, for the details of how specific clients work, please refer to the respective [client documentation](../../client-libraries.mdx).
+
+
+- **Avoid conflicts**: This can be done by
+  implementing application-level logic to prevent concurrent transactions from
+  modifying the same data points. This strategy can significantly reduce the
+  likelihood of encountering errors. It will also lead to better performance 
+  since the system is not spending time resolving conflict
 
 Handling serialization errors effectively is essential for maintaining a smooth
 user experience and ensuring the reliability of your application. Implementing
 robust error handling and conflict avoidance mechanisms can mitigate the impact
 of these errors.
 
+While some client drivers may handle serialization errors by retrying transactions, 
+developers should not solely rely on this mechanism. Always include comprehensive error  
+handling in your application to address cases where the error persists beyond the retry logic. 
+
```

Comment:
```suggestion
While some client drivers may handle serialization errors by retrying transactions, 
developers should not solely rely on this mechanism. Always include comprehensive error handling in your application to address cases where the error persists beyond the retry logic. 
```

Reply:
There was an extra empty space here, causing part of the line to skip into new line (check local deployment)